### ISSUE TITLE
Trained as what you said but got bad performance

### ISSUE THREAD

**Issue by shuluoshu** [laughtervv/SGPN#8]
Hi, @laughtervv 
 I trained as what you suggested, first pretrain the segmentation model, then use only SIM loss for the first 5 epoch and finally use the total loss. The total loss dropped to 20 and stays after 100 epochs. However, when I do the prediction, the results are bad: mAP : 0.004, other data is also far away from what you gives in your paper. I wonder what happens. The loss of yours can be as low as how much? 

Thanks.

**Comment by laughtervv** [comment]
0.004 looks like a bug or so. Can you try the pretrained model?
Can I see your training log and 'pergroup_thres.txt' and 'mingroupsize.txt'?

**Comment by shuluoshu** [comment]
[log.txt](https://github.com/laughtervv/SGPN/files/2204071/log.txt)

The above is `log.txt`,  and next is the `pergroup_thres.txt`  and `mingroupsize.txt`

[pergroup_thres.txt](https://github.com/laughtervv/SGPN/files/2204075/pergroup_thres.txt)


[mingroupsize.txt](https://github.com/laughtervv/SGPN/files/2204078/mingroupsize.txt)
  

Thanks @laughtervv

**Comment by laughtervv** [comment]
There's something wrong with your 'pergroup_thres.txt'. 
Here are the files I generated. Please give it a try.
[pergroup_thres.txt](https://github.com/laughtervv/SGPN/files/2207212/pergroup_thres.txt)
[mingroupsize.txt](https://github.com/laughtervv/SGPN/files/2207213/mingroupsize.txt)

**Comment by jianuo1128** [comment]
@laughtervv @shuluoshu 
Hello author, I used your pre-trained model, and when running the valid.py, I encountered the following nan problem, and the two txt files generated are as follows
![e99c38c1f60ba31cb5da185b630d971](https://github.com/user-attachments/assets/1903241e-60af-49b3-89f5-54aa33cce5ad)
![image](https://github.com/user-attachments/assets/478e2576-431b-4625-a0c0-4ae0ae3523a5)
![image](https://github.com/user-attachments/assets/2fbb3dfe-98bc-4aa8-8b75-b376c4a56e8b)
